openlibrary/core/sponsorships.py

```diff
@@ -175,3 +176,67 @@ def qualifies_for_sponsorship(edition):
         })
     })
     return resp
+
+def summary():
+    """
+    Provides data model (finances, state-of-process) for the /admin/sponsorship stats page.
+
+    Screenshot:
+    https://user-images.githubusercontent.com/978325/71494377-b975c880-27fb-11ea-9c95-c0c1bfa78bda.png
+    """
+    from internetarchive import search_items
+    params = {'page': 1, 'rows': 500}
+    fields = ['identifier','est_book_price','est_scan_price', 'scan_price',
+              'book_price', 'repub_state', 'imagecount', 'title',
+              'openlibrary_edition']
+    q = 'collection:openlibraryscanningteam'
+    config = dict(general=dict(secure=False))
+
+    # XXX Note: This `search_items` query requires the `ia` tool (the
+    # one installed via virtualenv) to be configured with (scope:all)
+    # privileged s3 keys.
+    s = search_items(q, fields=fields, params=params, config=config)
+
+    items = list(s)
+
+    # Construct a map of each state of the process to a count of books in that state
+    STATES = ['Needs purchasing', 'Needs digitizing', 'Needs republishing', 'Complete']
+    statuses = OrderedDict((status, 0) for status in STATES)
+    for book in items:
+        if not book.get('book_price'):
+            book['status'] = STATES[0]
+            statuses[STATES[0]] = statuses.get(STATES[0], 0) + 1
```

Comment:
Since you've init-ed the dict, these aren't needed anymore :)

```suggestion
            statuses[STATES[0]] += 1
```